src/main/java/ru/spbstu/dto/AnnotatedVariantList.java

```diff
@@ -0,0 +1,19 @@
+package ru.spbstu.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import ru.spbstu.model.Variant;
+
+import java.util.List;
+
+public class AnnotatedVariantList {
+    @JsonProperty("variantAnnotations")
+    public List<Variant> variants;
+
+    public List<Variant> getVariants() {
+        return variants;
+    }
+
+    public void setVariants(List<Variant> variants) {
+        this.variants = variants;
```

Comment:
Please use `@Data` annotation instead of manual writing getters and setters.
And do we really need setter here? It is always better to use builder or constructor for field initialization.

Reply:
Used @Value annotation instead of @Data annotation to guarantee immutability of VariantListDto object.